.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Hmmm this file shouldn't be committed. Can we remove it and add .vscode to the .gitignore?